### PR TITLE
Add BOXBOY!

### PR DESCRIPTION
**Pull request by gianveloxbr**
Adds BOXBOY!.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/boxboy/game.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "BOXBOY!"
+description = "BoxBoy!, is a role-playing video game developed by Silicon Studio for the Nintendo 3DS handheld video game console."
```
Comment (CodingKoopa) [inline games/boxboy/game.dat]: > BoxBoy!

This should be consistent with the `title` field.

Comment (CodingKoopa) [inline games/boxboy/game.dat]: > BoxBoy!,

The comma here is unnecessary.

Comment (CodingKoopa) [inline games/boxboy/game.dat]: This description is a bit short, maybe add "You play as x" and/or "You do Y".

**Comment by gianveloxbr** [comment]
Updated as requested.

File: games/boxboy/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
+title = "0004000000154500"
+region = "usa"
+release_date = "2015-04-02"
+
+[[ testcases ]]
```
Comment (Leo626) [inline games/boxboy/game.dat]: Testcases aren't added here anymore. Should be submitted through Citra (Help -> Compatibility).

**Comment by Leo626** [comment]
Add the other region releases too.
* `eur` - `0004000000154600` - `2015-04-02`
* `jpn` - `0004000000126800` - `2015-01-15`

**Comment by gianveloxbr** [comment]
Updated.

**Pull request merged**